gravityflow/gravityflow
# Update namespaces in accordance with newly-established Gravity Forms 2.5 beta conventions

## Description
This PR updates all namespaces as introduced by the Gravity Forms conventions. Updates as follows:
1. The usage of Gravity Forms Settings API namespace has been updated from `\Rocketgenius\Gravity_Forms\Settings\` to `\Gravity_Forms\Gravity_Forms\Settings\`.
2. Checks for the existence of Settings API classes before loading them.
3. Top-level namespace of Gravity Flow has been updated from `Gravity_Flow` to `Gravity_Flow\Gravity_Flow`.

This branch will be merged after the release of Gravity Forms 2.5-beta-3.

## Testing instructions
1. Check with the current Gravity Forms 2.5 beta, to ensure everything works perfectly. Enable debugging to note any issues. One such reported on Support can be found here: [HS#15525](https://secure.helpscout.net/conversation/1347357883/15524?folderId=1113492#thread-3882936055).
2. Also check with the current stable release of Gravity Forms 2.4 to ensure nothing breaks there.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
Any addons using the Gravity_Flow namespace must be updated to `Gravity_Flow\Gravity_Flow`. Use the Gravity_Flow fields would now be done as: 
`use Gravity_Flow\Gravity_Flow\Settings\Fields;`

## Checklist:
